src/constants/fixed_bases.rs

```diff
@@ -19,8 +19,12 @@ pub mod value_commit_v;
 pub const ORCHARD_PERSONALIZATION: &str = "z.cash:Orchard";
 
 /// SWU hash-to-curve personalization for the value commitment generator
+/// TODO: should we change to "NOTE_TYPE_PERSONALIZATION"?
```

Comment:
I don't think so; this is still needed with its original value, and the name is still accurate.

Reply:
Yeah, make scene, removed it.